src/routing/Router.svelte

```diff
@@ -58,14 +59,20 @@
       // Issue #355: Update the probe here, whenever the path changes, to ensure
       // that clicks to the back/forward buttons work as expected.
       if (probeName) {
-        store.setField('probeName', probeName);
-        if ($probeSet) {
-          let newProbe = $probeSet.find((p) => p.name === probeName);
+        let newProbe;
+
+        // The canonical probe info fetch. (PSS)
+        getSearchResults(probeName).then((r) => {
+          newProbe = { ...r[0], loaded: true };
+          store.setField('probe', newProbe);
+          store.setField('probeName', probeName); // This is temporary: use store.probe.name
```

Comment:
I'm not sure I understand this comment. Was this meant to be addressed before putting up the PR?

Reply:
should be addressed before me merge to main yes, this PR is a draft